src/VerifiableCredential.ts

```diff
@@ -38,82 +39,124 @@ export interface VCDIVerifiableCredential extends LinkedDataObject {
   // Subsequent elements are either URLs for other contexts OR
   // inline context objects.
   // https://w3c.github.io/vc-data-model/#contexts
-  '@context': string[] | any
+  '@context': string[] | any;
 
   // https://w3c.github.io/vc-data-model/#identifiers
-  id?: string
+  id?: string;
 
   // https://w3c.github.io/vc-data-model/#types
-  type: Type
+  type: Type;
 
   // https://w3c.github.io/vc-data-model/#issuer
-  issuer: string | IssuerObject
+  issuer: string | IssuerObject;
 
-  // https://w3c.github.io/vc-data-model/#issuance-date
-  issuanceDate: string
+  // https://w3c.github.io/vc-data-model/#validity-period
+  validFrom?: string;
 
-  // https://w3c.github.io/vc-data-model/#expiration
-  expirationDate?: string
+  // https://w3c.github.io/vc-data-model/#validity-period
+  validUntil?: string;
 
   // https://w3c.github.io/vc-data-model/#credential-subject
-  credentialSubject: CredentialSubject
+  credentialSubject: CredentialSubject | CredentialSubject[];
 
   // https://w3c.github.io/vc-data-model/#status
-  credentialStatus?: CredentialStatus
+  credentialStatus?: CredentialStatus;
 
   // https://w3c.github.io/vc-data-model/#data-schemas
-  credentialSchema?: CredentialSchema
+  credentialSchema?: CredentialSchema | CredentialSchema[];
+
+  // https://w3c.github.io/vc-data-model/#integrity-of-related-resources
+  relatedResource?: RelatedResource[];
```

Comment:
```suggestion
  relatedResource?: RelatedResource | RelatedResource[];
```

Reply:
I wanted to put this too, but I couldn't find an example of a single one anywhere. Plus, I found `array of objects` in the spec:

> To validate that a resource referenced by a [verifiable credential](https://w3c.github.io/vc-data-model/#dfn-verifiable-credential) is the same at verification time as it is at issuing time, an implementer MAY include a property named relatedResource that stores an array of objects that describe additional integrity metadata about each resource referenced by the [verifiable credential](https://w3c.github.io/vc-data-model/#dfn-verifiable-credential). If relatedResource is present, there MUST be an object in the array for each remote resource for each context used in the verifiable credential.

If you think it's useful to have anyway, I can add it.